shared/chat/inbox/row/bottom-line.js

```diff
@@ -0,0 +1,140 @@
+// @flow
+import React from 'react'
+import {Text, Markdown, Box} from '../../../common-adapters'
+import {
+  globalStyles,
+  globalColors,
+  globalMargins,
+  styleSheetCreate,
+  collapseStyles,
+  lineHeight,
+} from '../../../styles'
+import {isMobile} from '../../../constants/platform'
+
+type BottomLineProps = {
+  backgroundColor: ?string,
+  participantNeedToRekey: boolean,
+  showBold: boolean,
+  snippet: ?string,
+  subColor: ?string,
+  youNeedToRekey: boolean,
+}
+
+function BottomLine(props: BottomLineProps) {
```

Comment:
actually we should keep these as PureComponent as they're optimized to not rerender w/ shallowcompare. react hasn't yet made a pass at optimizing this sadly